docs/CSS12-user_prefs.md

```diff
@@ -306,22 +304,6 @@ Required flag: `--USER__advancedSettings: readium-advanced-on`
 
 Override class: User settings (should be applied by any means necessary)
 
-#### Type scale
```

Comment:
Type scale is fully deprecated, there's nothing to replace it? I guess it comes from using `zoom`?

Reply:
Well spotted, it should have been removed from the docs – it was only partly. 

So that was indeed tied to the font-size normalise we were using, as it was part of the font-size overrides we were applying to body copy elements (`h1`, `h2`, etc.).

Since `zoom` is global, we’re losing that ability.

Reply:
Alright, I'm not sure it was used much anyway by integrators. We certainly didn't show it in the user preferences. Having a consistent font size is much more important to our users.